test/test.http

```diff
@@ -20,3 +20,39 @@ GET http://localhost:8080/styles/styles.css
 ###
 
 GET http://localhost:8080/execute/testScriptName
+
+####
+// bad test do not test
```

Comment:
You've add the quotations to this test, so it will work. You can remove the comment about it being bad.